pandas/tests/arrays/sparse/test_libsparse.py

```diff
@@ -197,7 +197,8 @@ def _check_correct(a, b, expected):
             assert (result.equals(expected))
 
         def _check_length_exc(a, longer):
-            pytest.raises(Exception, a.intersect, longer)
+            with pytest.raises(Exception, match="<enter message here>"):
+                a.intersect(longer)
```

Comment:
didn't convert this on first pass since test is skipped on windows. will update from ci.